src/plugins/discover/public/application/components/loading_spinner/index.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export { LoadingSpinner, createLoadingSpinnerDirective } from './loading_spinner';
```

Comment:
You only export `LoadingSpinner` for tests right? And you do it directly from `loading_spinner.tsx` file so the export for `LoadingSpinner` from `index` is not used anywhere... Maybe you could either export from here or remove the export 🤔

Reply:
No I export both the directive and the component and use it also to the angular part so I prefer to have them both in index.ts to be more clear

Reply:
Well I haven't finished with this PR yet :D